frontend/src/reports/ui/VoteDotOrgBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import HowToVoteIcon from '@mui/icons-material/HowToVote'
+import HetNotice from '../../styles/HetComponents/HetNotice'
+import type { DataTypeId } from '../../data/config/MetricConfig'
+import { useParamState } from '../../utils/hooks/useParamState'
+import { VOTE_DOT_ORG_PARAM_KEY } from '../../utils/urlutils'
+import HetLinkButton from '../../styles/HetComponents/HetLinkButton'
+
+export const LIFELINE_IDS: DataTypeId[] = ['suicide', 'gun_violence_suicide']
```

Comment:
This is also defined in `LifelineAlert.tsx`. Is it needed here?

Reply:
nice catch!